cms/static/sass/bootstrap/_variables.scss

```diff
@@ -28,7 +28,6 @@ $studio-border-color: palette(grayscale, back) !default;
 $studio-label-color: palette(grayscale, black) !default;
 $studio-active-color: palette(primary, base) !default;
 $studio-preview-menu-color: #c8c8c8 !default;
-$success-color: palette(success, accent) !default;
```

Comment:
why did this one disappear? is it elsewhere?

Reply:
It is no longer necessary as we use Bootstrap's theme's success color everywhere.